libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Conway/Gov.hs

```diff
@@ -158,15 +158,13 @@ allGASAndChildInTree t k =
 
 wellFormedChildren ::
   IsConwayUniv fn =>
-  Term fn (StrictMaybe (GovActionId StandardCrypto)) ->
   Term fn ProposalTree ->
   Pred fn
-wellFormedChildren root rootAndTrees =
+wellFormedChildren rootAndTrees =
   match rootAndTrees $ \root' trees ->
-    [ assert $ root ==. root' -- The root matches the root given in the environment
```

Comment:
What happened to this constraint?

Reply:
I removed `GovRelation` from `GovEnv`, since it was the exact same value as the `GovRelation` in the state. Now this constraint became redundant since we don't have to keep these two values in sync anymore.

Reply:
This implies that we assume that `proposalsApplyEnactment` is correct, but I suppose even though we eliminate the use of `prevGovActionIds` from the `Environment`, the correctness will still be tested via traces over the `EPOCH` rule.